intellij-plugin/build.gradle.kts

```diff
@@ -24,8 +25,8 @@ repositories {
 
 group = properties("pluginGroup")
 
-// Use the global version defined in the root project + snapshot suffix if from the CI
-version = properties("VERSION_NAME") + if (isSnapshotBuild()) ".${properties("snapshotVersion")}" else ""
+// Use the global version defined in the root project + dedicated suffix if building a snapshot from the CI
+version = properties("VERSION_NAME") + if (isSnapshotBuild()) ".${SimpleDateFormat("YYYY-MM-dd").format(Date())}" else ""
```

Comment:
Could we use the git commit instead or does that need to increment monotonically?

Reply:
I'm pretty sure the publication date is used to check if a more recent version is used, so the git commit should work - but are you sure that'd be better? A date may be a bit more straightforward/may ease communication if someone reports an issue?

Reply:
Why not both? :)

Reply:
(cut the sha1 at 7 chars, like GH is doing to avoid adding too much entropy in this world?)

Reply:
Ah both, good idea :) 👍